packages/rpc-core/src/rpc-methods/__tests__/get-recent-performance-samples-test.ts

```diff
@@ -0,0 +1,25 @@
+// import { createHttpTransport, createJsonRpc } from '@solana/rpc-transport';
+// import type { Rpc } from '@solana/rpc-transport/dist/types/json-rpc-types';
+import fetchMock from 'jest-fetch-mock-fork';
+// import { createSolanaRpcApi, SolanaRpcMethods } from '../index';
+
+describe('getRecentPerformanceSamples', () => {
+    // let rpc: Rpc<SolanaRpcMethods>;
+    beforeEach(() => {
+        fetchMock.resetMocks();
+        fetchMock.dontMock();
+        // rpc = createJsonRpc<SolanaRpcMethods>({
+        //   api: createSolanaRpcApi(),
+        //   transport: createHttpTransport({ url: 'http://127.0.0.1:8899' }),
+        // });
+    });
```

Comment:
If we can't do it, just delete all of this. By the time we can, the test infra might have changed.